.env.example

```diff
@@ -0,0 +1,75 @@
+ANNUAL_COST_EMPLOYEE=650000
+BUSINESS_HOURS_YEAR=2080
+COMPOSE_PROJECT_NAME=dispatch
+DATABASE_CREDENTIALS=creds
+DATABASE_HOSTNAME=postgres
+DATABASE_NAME=dispatch
+DATABASE_PORT=5432
+DISPATCH_HELP_EMAIL=help@example.com
+DISPATCH_HELP_SLACK_CHANNEL=#general
+INCIDENT_DAILY_SUMMARY_ONCALL_SERVICE_ID=None
+INCIDENT_DOCUMENT_INVESTIGATION_SHEET_ID="INCIDENT_DOCUMENT_INVESTIGATION_SHEET_ID"
+INCIDENT_FAQ_DOCUMENT_ID="INCIDENT_FAQ_DOCUMENT_ID"
+INCIDENT_NOTIFICATION_CONVERSATIONS=""
+INCIDENT_NOTIFICATION_DISTRIBUTION_LISTS=""
+INCIDENT_PLUGIN_CONTACT_SLUG="slack-contact"
+INCIDENT_PLUGIN_CONVERSATION_SLUG="slack-conversation"
```

Comment:
A lot/most of these have defaults, so they shouldn't need to be specified explicitly (just overridden if necessary).

Reply:
See: https://github.com/Netflix/dispatch/blob/develop/docker/.env.example#L2 all the options that are actually required.

Reply:
Yep! I included them mostly as documentation. It wasn't intuitive since I was configuring in dispatch-docker but had to go look at another repo to see what the config variables were.

The other confusing issue for me was not being able to get the app to boot without errors without having _something_ supplied for all of these.

But I also did this before asking about https://github.com/Netflix/dispatch/issues/55

I think if I had read in the README that I needed to configure everything before trying to boot up, I wouldn't have made that effort.

Reply:
Agreed, I will add a blub to the docs it to: https://hawkins.gitbook.io/dispatch/installation